src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/PolicyControllerTest.cs

```diff
@@ -382,6 +382,42 @@ public async Task GetResourcePoliciesFromXacmlPolicies_NoOrg_TC08()
             AssertionUtil.AssertCollections(expectedResourcePolicyResponses, actualResourcePolicyResponses, AssertionUtil.AssertResourcePolicyResponseEqual);
         }
 
+        /// <summary>
+        /// Test case: Try GetResourcepolicies where a ResourcePolicy exists in blobstorage, but does not match the Org/App in the request
+        /// Expected: GetResourcepolicies returns a list of 
```

Comment:
what's the expected result?

Reply:
Test Case can also be a bit more specific in that:
"... but resource specification in the xacml policy does not match ..."

Reply:
Fixed